design-components/examples/react-example/src/App.js

```diff
@@ -36,6 +58,24 @@ function App() {
   const [buttonTheme, setButtonTheme] = useState('primary');
   const [disableButton, setDisableButton] = useState(true);
 
+  const updatePages = (page) => {
+    console.log('clicked');
```

Comment:
We want to left this here?

Reply:
this is just on the demo page, for testing purposes, not public